boards/firechip/base-workloads/ubuntu-base.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name" : "ubuntu-base",
+  "guest-init": "initRepos.sh",
+  "rootfs-size": "16GB",
```

Comment:
This seems very big... can this be minimized? Or just set to the default?

Reply:
done